spec/jekyll_seo_tag_integration_spec.rb

```diff
@@ -30,6 +30,10 @@
     expect(output).to match(%r!Jekyll v#{version}!i)
   end
 
+  it "outputs JSON sorted by key" do
+    expect(json.strip).to eql("{\"@context\":\"https://schema.org\",\"@type\":\"WebPage\",\"url\":\"/page.html\"}")
```

Comment:
How about using `%()` or outer single-quotes to avoid escaping the JSON's double-quotes?

```suggestion
    expect(json.strip).to eql('{"@context":"https://schema.org","@type":"WebPage","url":"/page.html"}')
```